### ISSUE TITLE
Deployment in a new VPC gives version 0.999.0dev and not 1.4.0

### ISSUE THREAD

**Issue by nazbadat** [aws-quickstart/quickstart-apache-superset#32]
Hi there, 

Is there a way to upgrade superset after deploying via CloudFormation? I notice that the CF template to deploy to an existing VPC notes that the platform version should be 1.4.0, but the new VPC template file has no reference to a version being used. We want to upgrade to the latest version as there is no BigQuery database connection types included. Any clarification on this would be greatly appreciated!

Thanks
Naz

<img width="191" alt="Screenshot 2022-01-07 at 14 09 25" src="https://user-images.githubusercontent.com/64232584/148555637-174b98c7-0dca-4f20-b42e-3865c74ac129.png">

**Comment by yike5460** [comment]
Hi, the upstream sync up in on going, and I assume BigQuery support is what you need I such release?

**Comment by nazbadat** [comment]
@yike5460 Yes that is correct. @rainerledesma Unfortunately due to business needs of the tool, I cloned the repository and ran it locally using docker-compose, then added the necessary config to superset-config.py and it worked as expected. The version number still didn't change, but BigQuery was functional.

**Comment by yike5460** [comment]
1.4 is not mandatory for BigQuery, the dependency you need to install is https://github.com/googleapis/python-bigquery-sqlalchemy, with command
`pip install sqlalchemy-bigquery`

**Comment by yike5460** [comment]
@nazbadat v1.4.1 is now merged in official solution, kindly let me know your further feedback.
<img width="104" alt="image" src="https://user-images.githubusercontent.com/23544182/155433185-c4396fe7-721d-49a3-a0b9-0f44cd100a1b.png">
